CMakeLists.txt

```diff
@@ -19,80 +22,70 @@
 # CTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 # ########################################################################
 
-cmake_minimum_required( VERSION 3.11 )
+cmake_minimum_required(VERSION 3.11)
+project(hipblas-common VERSION 1.2.0 LANGUAGES CXX)
 
-# Build documentation
-option(BUILD_DOCS "Build documentation" OFF)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/rocm-cmake/share/rocmcmakebuildtools/cmake")
 
-# Consider removing this in the future
-# This should appear before the project command, because it does not use FORCE
-if( WIN32 )
-  set( CMAKE_INSTALL_PREFIX "${PROJECT_BINARY_DIR}/package" CACHE PATH "Install path prefix, prepended onto install directories" )
-else( )
-  set( CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended onto install directories" )
-endif( )
+include(ROCMInstallTargets)
+include(ROCMCreatePackage)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
 
-# This has to be initialized before the project() command appears
-# Set the default of CMAKE_BUILD_TYPE to be release, unless user specifies with -D.  MSVC_IDE does not use CMAKE_BUILD_TYPE
-if( NOT DEFINED CMAKE_CONFIGURATION_TYPES AND NOT DEFINED CMAKE_BUILD_TYPE )
-  set( CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel." )
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    if(WIN32)
+      set(CMAKE_INSTALL_PREFIX "C:/hipSDK" CACHE PATH "Install path prefix" FORCE)
+    else()
+      set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix")
```

Comment:
Just noticed this doesn't have `FORCE`, which it should. We're already guaranteed to not be overriding a user-provided value, and without `FORCE` this actually won't have any effect as the `CMAKE_INSTALL_PREFIX` cache variable has already been initialized.